source/fundamentals/logging.txt

```diff
@@ -56,9 +56,9 @@ level:
 Filter on a Specific Class
 --------------------------
 
-You can set the Logger to only produce log messages that are reported by
-specific classes. The following example demonstrates how to log messages from
-only the ``Db`` class.
+You can set the Logger to only produce log messages generated by specific
+classes. The following example demonstrates how to log messages from the
```

Comment:
I feel like "only" is actually quite valuable here. What do you think?

Reply:
Ok. I added it to the end to make it slightly less ambiguous what it was modifying.